composer.json

```
@@ -11,14 +11,14 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 | ^8.0",
         "php-http/httplug": "^2.0",
         "psr/http-client": "^1.0",
         "guzzlehttp/guzzle": "^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.4",
-        "php-http/client-integration-tests": "^2.0"
+        "phpunit/phpunit": "^9.3",
```

```suggestion
        "phpunit/phpunit": "^8.0|^9.3",
```

would php 7.2 build if we allow an older phpunit? or is there another problem then?

---

a very good point. yes, phpunit8 is also compatible. I just added it and the tests passed :+1: